schemas/context/namespace.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+   "meta:license":[
+      "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+      "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+      "you may not use this file except in compliance with the License. You may obtain a copy",
+      "of the License at https://creativecommons.org/licenses/by/4.0/"
+   ],
+   "$id":"https://ns.adobe.com/xdm/context/namespace",
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "title":"Namespace",
+   "type":"object",
+   "description":"The Datasource acts as a namespace or unique identifier associated with a collection of data. Each EndUserID is associated with a given Datasource. A user can create and obtain information about a Datasource during the solution onboarding.\n",
```

Comment:
Inconsistency between "Namespace" in the title and "Datasource" in the description.

Reply:
Good point. This is a left over description of when this schema was created from the Datasource schema. Both the Namespace and Datasource schema are identical except they have different use cases. One is used to represent the Namespace of a given Identity (aka user identifier). The other is used to represent the Datasource of a source of data.

Reply:
I think then it makes sense to unify this notion of a "governor of data", as we encounter it in various places:
- where data points are coming from
- generator of IDs
- resolver of IDs
- decider of segment membership
- etc.

I like the term "data source" better than "namespace" because it is more specific and less ambiguous, but it might not adequately capture all meanings. "Realm" has a poetic quality to it, but is even less clear than namespace.

Reply:
There is a recurring pattern used here to make the id spaces unique in systems that are loosely coupled with super high efficiency. While the pattern is the same, the id spaces being made unique are very different. End Users, Segments, Data producers, Merging/stitching functions. The concept of "governor of data" is interesting. Unifying the naming has proven elusive.

Reply:
That's why I strongly distinguish between stuff that's covered by XDM, which is using URIs as `@id` and stuff that's ruled by implementation details, which can be any string in any `xdm:id` attribute.

The goal of the "Namespace/Data Source/Realm/Governor" is to identify which system is providing the specific implementation. Keeping track of a list of systems where data is originating is the least we can do.

Reply:
I think we all agree with this, and is what we were trying to do. The Identity (which contains the id and namespace of that ID) is always under a specific namespaced field/implementation - "https://ns.adobe.com/experience/analytics", "https://ns.adobe.com/experience/mcID", etc.

Reply:
Action item for @jwen-adobe: 
- [ ] unify this as "Namespace" for now. Most of the ID-stuff discussed here belongs in another thread in the same PR.

Reply:
@trieloff @cmathis done.